scout/server/blueprints/cases/controllers.py

```diff
@@ -388,6 +390,48 @@ def coverage_report_contents(store, institute_obj, case_obj, base_url):
     return coverage_data
 
 
+def mt_coverage_stats(individuals, ref_chrom="21"):
+    """Send a request to chanjo report endpoint to retrieve MT vs autosome coverage stats
+
+    Args:
+        individuals(dict): case_obj["individuals"] object
+        ref_chrom(str): reference chromosome (1-22)
+
+    Returns:
+        coverage_stats(dict): a dictionary with mean MT and autosome transcript coverage stats
+    """
+    coverage_stats = {}
+    ind_ids = []
+    for ind in individuals:
+        ind_ids.append(ind["individual_id"])
+
+    # Prepare complete url to Chanjo report chromosome mean coverage calculation endpoint
+    cov_calc_url = url_for("report.json_chrom_coverage", _external=True)
+    # Prepare request data to calculate mean MT coverage
+    data = dict(sample_ids=",".join(ind_ids), chrom="MT")
+    # Send POST request with data to chanjo endpoint
+    resp = requests.post(cov_calc_url, json=data)
+    mt_cov_data = json.loads(resp.text)
+
+    # Change request data to calculate mean chr 21 coverage
+    data["chrom"] = str(ref_chrom)  # convert to string if an int is provided
+    # Send POST request with data to chanjo endpoint
+    resp = requests.post(cov_calc_url, json=data)
+    ref_cov_data = json.loads(resp.text)  # mean coverage over the transcripts of ref chrom
+
+    for ind in ind_ids:
+        if not (mt_cov_data.get(ind) and ref_cov_data.get(ind)):
+            continue
+        coverage_info = dict(
+            mt_coverage=round(mt_cov_data[ind], 2),
+            autosome_cov=round(ref_cov_data[ind], 2),
+            mt_autosome_ratio=round(mt_cov_data[ind] / ref_cov_data[ind], 2),
```

Comment:
```suggestion
            mt_autosome_ratio=round(mt_cov_data[ind] / ref_cov_data[ind] * 2, 2),
```
But you knew that already! 😊

Reply:
> Yes, never 19 or 17!
> I would personally feel a tad irrationally uncomfortable with 13, since trisomy of is viable, though very rare. 11 or 15 aren't that different I guess, and have a little higher / closer GC% to MT?

I've seen many genomics profiles with somatic trisomy 15 in elderly in my previous job, I would exclude it as well.